cmd/odf/restore/crds.go

```diff
@@ -1,12 +1,63 @@
 package restore
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
 	pkgrestore "github.com/rook/kubectl-rook-ceph/pkg/restore"
 	"github.com/spf13/cobra"
 )
 
+// groupVersions defines the supported CRD groups and their corresponding API versions.
+var groupVersions = map[string]string{
+	"ocs.openshift.io":       "v1",
+	"ceph.rook.io":           "v1",
+	"csi.ceph.io":            "v1beta1",
+	"odf.openshift.io":       "v1alpha1",
+	"noobaa.io":              "v1alpha1",
+	"csiaddons.openshift.io": "v1alpha1",
+}
+
+// keys returns the keys of a string map. It is used to print out supported group names.
+func keys(m map[string]string) []string {
```

Comment:
```suggestion
func groupNameKeys(m map[string]string) []string {
```
Let's have name related to their work

Reply:
done